src/transforms/remap.rs

```diff
@@ -72,8 +72,12 @@ impl FunctionTransform for Remap {
         emit!(RemapEventProcessed);
 
         let mut runtime = Runtime::default();
+        let result = match event {
+            Event::Log(ref mut event) => runtime.execute(event, &self.program),
+            Event::Metric(ref mut event) => runtime.execute(event, &self.program),
+        };
```

Comment:
Another minor nit that we should be able to keep `runtime.execute` out of the pattern matching.